src/inspector/value-mirror.cc

```diff
@@ -1433,8 +1433,22 @@ bool ValueMirror::getProperties(v8::Local<v8::Context> context,
     }
   }
 
+  // [RUN-3149] we're only interested in a subset of the properties on this
+  // object. In order to make sure we get enough own properties to satisfy
+  // expected behavior for objects, always request 100k keys.  we will
+  // later prune returned properties down to the originally requested size.
+  v8::KeyIterationParams subsetIterationParams(100000, 0);
```

Comment:
Can we avoid unnamed constants?
(Oh boy it will be nice once we can easily code-share between chromium and V8 😆)